docs/html-templates.md

```diff
@@ -0,0 +1,40 @@
+# Html templates
+
+### Behavior
+
+* Templates are automatically recompiled in development when the file
```

Comment:
You might note that you have to do a hard refresh, considering some browsers cache webpages.